pages/index.tsx

```diff
@@ -8,29 +8,44 @@ import IntroSection from '../components/IntroSection';
 import JoinUs from '../components/JoinUs';
 import Services from '../components/Services';
 import Solutions from '../components/Solutions';
-import '@fontsource/montserrat/600.css';
-import '@fontsource/montserrat/700.css';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 export default function Home() {
+  const [mount, setMount] = useState(false);
+
+  useEffect(() => {
+    setMount(!mount);
+  }, []);
+
```

Comment:
This completely cancels server-side rendering, which cancels SEO benefits. Why is it needed? Can it be done without?

Reply:
Yeah. it can be done without it.